src/index.css

```
@@ -1057,16 +1068,16 @@
   }
 
   .message-action-btn:hover:not(:disabled) {
-    @apply bg-white/10 text-input-text;
+    @apply bg-black/5 dark:bg-white/10 text-input-text;
   }
 
   .message-reaction-chip {
-    @apply inline-flex items-center gap-1 rounded-full border border-white/10 bg-white/5 px-2 py-0.5 text-xs text-input-text transition;
+    @apply inline-flex items-center gap-1 rounded-full border border-black/5 dark:border-white/10 bg-black/5 dark:bg-white/5 px-2 py-0.5 text-xs text-input-text transition;
     box-shadow: none;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)